packages/lesswrong/lib/collections/users/custom_fields.js

```diff
@@ -53,6 +54,34 @@ Users.addField([
     }
   },
 
+  {
+    fieldName: 'commentSorting',
+    fieldSchema: {
+      type: String,
+      optional: true,
+      viewableBy: ['guests'],
+      insertableBy: ['members'],
+      editableBy: ['members'],
+      order: 65,
+      control: "select",
+      form: {
+        options: function () { // options for the select form control
+          let commentViews = [
+            {value:'postCommentsTop', label: 'magical algorithm'},
+            {value:'postCommentsNew', label: 'most recent'},
+            {value:'postCommentsOld', label: 'oldest'},
```

Comment:
Ideally we would have this view list factored out somewhere, instead of having the concerns duplicated. I don't think this is necessary, but you should at least add a `TODO: ` in the code.